### PR TITLE
feat(users.credentials): generate s3 credentials

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/high-perf-storage`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-7050
| License          | BSD 3-Clause

## Description

Generate user s3 credentials

### CODE REVIEW

**Comment by varun257** [comment]
Need approval for this
This was reviewed at https://github.com/ovh/manager/pull/5275 but wrongfully merged to develop.

**Review by ganeshkumar1989 — APPROVED** [review]

**Review by frenautvh — APPROVED** [review]

File: packages/manager/modules/pci/src/components/project/users/users.html [hunk 1]
```diff
@@ -133,6 +133,16 @@
                     data-translate="pci_projects_project_users_modify_password_label"
                 ></span>
             </oui-action-menu-item>
+
+            <oui-action-menu-item
+                data-on-click="$ctrl.generateCredentials($row)"
+                data-disabled="$ctrl.isActionDisabled()"
```
Comment (mohammed-zahaf) [inline packages/manager/modules/pci/src/components/project/users/users.html:139]: i think name is to generic, may be we can rename it.
```suggestion
                data-disabled="$ctrl.canGenerateCredentials()"
```

Comment (varun257) [reply packages/manager/modules/pci/src/components/project/users/users.html:139]: Hi @mohammed-zahaf , this is reused for all actions based on user status of account. Hence its not been changed. I am reusing the existing generic function here.

Comment (mohammed-zahaf) [reply packages/manager/modules/pci/src/components/project/users/users.html:139]: Ok got it

**Review by mohammed-zahaf — APPROVED** [review]

**Pull request merged**